backend/src/connectors/authorisation/base.ts

```diff
@@ -108,6 +116,16 @@ export class BasicAuthorisationConnector {
     )
   }
 
+  async responses(user: UserInterface, response: ResponseDoc, action: ResponseActionKeys): Promise<Response> {
+    if (action === ResponseAction.Update && toEntity('user', user.dn) !== response.user) {
+      return { id: user.dn, success: false, info: 'Only the author can update a comment' }
+    }
+    return {
```

Comment:
We should probably make sure that if we're creating a response, the user has access to the 'access request' they're creating the response against.